### ISSUE TITLE
[N7 Account Settings] Create Pronouns page

### ISSUE THREAD

**Issue by Beamanator** [Expensify/App#11576]
Tracking issue: https://github.com/Expensify/Expensify/issues/182122

Detailed section of doc here: https://docs.google.com/document/d/17akqrbC-w8FqEoDLvcLmrX6iw4uaVrT_xlwAl-IhEzE/edit#heading=h.5n8w70bpe690

Note: This can only be implemented after `UpdatePronouns` command has been built - see https://github.com/Expensify/Expensify/issues/238724

**Comment by Beamanator** [comment]
On hold for WhatsApp Quality

**Comment by Beamanator** [comment]
Still on hold for now, might pick this up next week as discussions from here (https://github.com/Expensify/App/issues/11795) are indicating this could be useful to implement sooner rather than later

**Comment by Beamanator** [comment]
Still on hold, need to discuss if the dropdown should be push-to-page to be consistent with other dropdowns (like new timezone selector)

**Comment by Beamanator** [comment]
Agreed [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1666978331169979?thread_ts=1666881832.702589&cid=C01GTK53T8Q) that we're going with the current design of the Pronouns page in the doc

**Comment by Beamanator** [comment]
FYI we now may go with the current Pronouns design & open a new, separate project for redesigning the page & deciding what kind of "custom pronoun selector / creator" we want to implement ([discussion](https://expensify.slack.com/archives/C01GTK53T8Q/p1668109710032789?thread_ts=1666881832.702589&cid=C01GTK53T8Q))

**Comment by Beamanator** [comment]
Latest, agreed-upon plan is here: https://expensify.slack.com/archives/C01GTK53T8Q/p1668468842096049?thread_ts=1666881832.702589&cid=C01GTK53T8Q

**Comment by thesahindia** [comment]
@Beamanator, C+ compensation is pending on this one.

**Comment by Beamanator** [comment]
Hey @mateocole can you pay out @thesahindia for his work reviewing the related internal PR? 👍

**Comment by Beamanator** [comment]
(reopening till payment has been completed)

**Comment by thesahindia** [comment]
> hey @Beamanator from reading the [S/O](https://stackoverflow.com/c/expensify/questions/7676) is there an upwork associated with the PR review

We will need to create one (no job right now)

If I am not wrong we just need to add external label and melvin will post the upwork job link on this issue.

**Comment by Beamanator** [comment]
@mateocole yes as you mentioned, we'll need an Upwork job for this. And interesting point @thesahindia , I know that automation is new, I don't necessarily like the idea of needing to add `External` label since this will also assign a C+ (which isn't needed for this issue) right? I'll ask internally, I think since all of this is new we'll eventually come up with better automation for this exact case :D

**Comment by thesahindia** [comment]
Yeah, makes sense!

**Comment by thesahindia** [comment]
Bump @mateocole for upwork

**Comment by mateocole** [comment]
Okay here is the job!
https://www.upwork.com/ab/applicants/1598482099900215296/job-details

**Comment by thesahindia** [comment]
@mateocole, I have applied and increased the job price since compensation for internal PR review is $1000 ([slack post](https://expensify.slack.com/archives/C01GTK53T8Q/p1668726522584349), please see the 4th point)

**Comment by Beamanator** [comment]
Hmm it looks like @thesahindia you reviewed on Nov 15 ([here](https://github.com/Expensify/App/pull/12301#issuecomment-1315853402)) but the update to increase C+ bounty to $1000 was made on [Nov 18](https://expensify.slack.com/archives/C01GTK53T8Q/p1668726522584349)... I'll leave it up to @mateocole / @michaelhaxhiu to decide, I'm just pointing out date diff :D

**Comment by thesahindia** [comment]
Oh correct, I didn't check the date 😅

**Comment by mateocole** [comment]
Hey @thesahindia, would you be able to withdraw the proposal at the current rate, and go back to the Nov 15 rate?

**Comment by thesahindia** [comment]
@mateocole, I have changed the terms.

**Comment by thesahindia** [comment]
Accepted, thanks!

**Comment by mateocole** [comment]
Okay, payment sent!

**Comment by thesahindia** [comment]
Let's close this issue.
